invenio_app_ils/records/jsonresolver/item_loan.py

```diff
@@ -26,6 +26,8 @@ def loan_for_item_resolver(item_pid):
         # remove the `item` field to avoid circular dependencies
         if loan.get("item"):
             del loan["item"]
+        if loan.get("$schema"):
```

Comment:
Is this fixing a bug?

Reply:
No, it was just to clean `$schema` from the resolved data since it's not needed.